sql_metadata/generalizator.py

```diff
@@ -0,0 +1,106 @@
+"""
+Module used to produce generalized sql out of given query
+"""
+import re
+from typing import List, Optional
+
+import sqlparse
+
+
+class Generalizator:
+    """
+    Class used to produce generalized sql out of given query
+    """
+
+    def __init__(self, sql: Optional[str] = None):
```

Comment:
Why is `sql` argument optional? What's the point of passing `None` here?

Reply:
In practice probably not very useful 🙈 
Since you can: `assert Parser("").tokens == []`

I assumed that that one should also pass: `assert Parser().tokens == []`

Like if you get the queries from somewhere and even if the query is None you can parse them all in a loop but that's a little far stretched 🤷

Reply:
Then let's default to an empty string instead of `None`:

```suggestion
    def __init__(self, sql: str = ""):
```